src/client/dispatch.rs

```diff
@@ -732,5 +732,14 @@ async fn handle_event(
                 event_handler.webhook_update(context, event.guild_id, event.channel_id).await;
             });
         },
+        #[cfg(feature = "unstable_discord_api")]
+        #[cfg_attr(docsrs, doc(feature = "unstable_discord_api"))]
```

Comment:
The `cfg_attr` attribute should only be placed on structs, fields, functions, consts, and statics. Anything that will be displayed in the documentation to be precise. While it does no harm, the attribute has no effect here, since this won't be displayed in the documentation (It's an arm of a `match`). What matters here is `#[cfg(feature = "unstable_discord_api")]`, as that is what is actually needed to prevent compilation if the feature is disabled.